README.md

```diff
@@ -245,12 +143,102 @@ When these are obtained, don't forget to update [config_terminal.py](/config_ter
 
 Note that it is not necessary to have a valid Alpha Vantage key to get daily OHLC values.
 
-## Disclaimer <a name="Disclaimer"></a>
+### Usage
 
-"A few things I am not. I am not a cat. I am not an institutional investor, nor am I a hedge fund. I do not have clients and I do not provide personalized investment advice for fees or commissions." DFV
+Start by loading a ticker of interest:
+```
+load -t GME
+```
+The menu will expand to all its menus since a ticker has been loaded.
 
-## Support <a name="Support"></a>
+View the historical data of this stock:
+```
+view
+```
+Apply some technical indicators to the ticker, starting at 2020-06-04.
+Re-load ticker with starting time, this time:
+```
+load -t GME -s 2020-06-04
+```
+Enter in technical analysis menu with
+```
+ta
+```
+and run a SMA with:
+```
+sma
+```
+However, imagine that you wanted to change the length of the window because you don't want to go long but do a swing, and therefore a smaller window is necessary. Check what settings are available on the SMA command:
+```
+sma -h
+```
+Once that has been seen, set the parameters that you want after flagging them. In this case, to change length window to 10, we would have to do:
+```
+sma -l 10
+```
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+There are 3 main ways of contributing to this project.
+
+**Become a Contributor 🦍**
+
+Recommended if you bought the dip, and the share price keeps dipping. You may as well keep yourself busy while stonks go up.
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to your Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
```

Comment:
Good place to link to a contributor's document.

Reply:
I'd add on the top, right under the "Become a Contributor"